battery_parameter_test.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import Battery_Parameter_Handler
+
+class test_battery_current_ranges(unittest.TestCase):
+    
+  def test_battery_range(self):    
+    self.assertTrue(Battery_Parameter_Handler.process_information(["Charge_rate" ,"Temperature"],"Console"))    
```

Comment:
This test will pass even if `process_information` produces output in wrong range, with wrong params, or no output even - as long as it returns True. So it's not really a test.